### PR TITLE
64 test coverage

### LINKED ISSUE #64: Pump up the test coverage 💪🏻
### Description
Test models, test utils, test everything!

### Subtasks
- [ ] Test GraphQL type resolution

### PR DESCRIPTION
**Pull request by fredrikbw**
Added some more tests

Closes #64

### CODE REVIEW

File: Backend.csproj [hunk 1]
```diff
@@ -47,4 +47,10 @@
       <Folder Include="Models" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Reference Include="Moq, Version=4.16.0.0, Culture=neutral, PublicKeyToken=69f491c39445e920">
+        <HintPath>..\..\Users\fredr\.nuget\packages\moq\4.16.1\lib\netstandard2.1\Moq.dll</HintPath>
+      </Reference>
+    </ItemGroup>
+
```
Comment (FredrikAugust) [inline Backend.csproj:55]: Does it work without this?

File: BackendTests/UnitTests/MetAPIServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        private readonly MetAPIServiceMocked _metAPIServiceMocked;
+        private readonly Random _random;
+
+
+        public MetAPIServiceTests()
+        {
+            _metAPIServiceMocked = new MetAPIServiceMocked();
+            _random = new Random();
+            _lon = (float) _random.NextDouble() * (63.433f - 63.42f) + 63.42f;
+            _lat = (float) _random.NextDouble() * (10.4f - 10.38f) + 10.38f;
```
Comment (FredrikAugust) [inline BackendTests/UnitTests/MetAPIServiceTests.cs:25]:
```suggestion
            _lon = 63.42;
            _lat = 10.38;
```

Simplify the test by using hard-coded variables.

Alternatively, you could perform some [property based testing](https://medium.com/criteo-engineering/introduction-to-property-based-testing-f5236229d237) by having lat and lon be generated in the respective accepted ranges (-180->180 and -90->90).

File: Backend.csproj (hunk 1 above)
Comment (fredrikbw) [reply Backend.csproj:55]: Yes

File: BackendTests/UnitTests/MetAPIServiceTests.cs (hunk 2 above)
Comment (fredrikbw) [reply BackendTests/UnitTests/MetAPIServiceTests.cs]: Changed

**Comment by FredrikAugust** [comment]
I've added the `closing` keyword to this PR, as I believe the issue should be closed once this is merged.

The intention was to get some momentum on the testing, which this issue succeeds in.

**Review by Kpaubert — APPROVED** [review]
Cool beans, Fredde :)

**Pull request merged**